BLSQ/iaso
# IA-3449: Spelling corrections

Related JIRA tickets : IA-3449

## Self proofreading checklist


## Doc
- Tell us where the doc can be found (docs folder, wiki, in the code...)

## Changes

- Fix some spelling

## How to test

- check the corrections in the po file
- Go to login page, click "forgot password"
- switch language to french
- The spelling of "Retour à la connexion" should be correct

## Print screen / video
![Screenshot 2024-10-28 at 18 07 18](https://github.com/user-attachments/assets/d10e590d-7b5d-4fc7-ac82-e414ea42d1ce)
